test/Base_Tests/src/Data/Decimal_Spec.enso

```diff
@@ -663,6 +663,55 @@ add_specs suite_builder =
+            Decimal.new "12" . min (Decimal.new "13") . should_equal (Decimal.new "12")
+            Decimal.new "12" . min (Decimal.new "11") . should_equal (Decimal.new "11")
+            Decimal.new "-12" . min (Decimal.new "-13") . should_equal (Decimal.new "-13")
+            Decimal.new "-12" . min (Decimal.new "-11") . should_equal (Decimal.new "-12")
+
+            Decimal.new "12.1" . min (Decimal.new "12.3") . should_equal (Decimal.new "12.1")
+            Decimal.new "12.2" . min (Decimal.new "11.7") . should_equal (Decimal.new "11.7")
+            Decimal.new "-12.1" . min (Decimal.new "-12.3") . should_equal (Decimal.new "-12.3")
+            Decimal.new "-12.2" . min (Decimal.new "-11.7") . should_equal (Decimal.new "-12.2")
+
+            Decimal.new "0" . min (Decimal.new "-1") . should_equal (Decimal.new "-1")
+            Decimal.new "0" . min (Decimal.new "1") . should_equal (Decimal.new "0")
+            Decimal.new "-1" . min (Decimal.new "0") . should_equal (Decimal.new "-1")
+            Decimal.new "1" . min (Decimal.new "0") . should_equal (Decimal.new "0")
+
+            Decimal.new "12E73" . min (Decimal.new "13E60") . should_equal (Decimal.new "13E60")
+            Decimal.new "12E73" . min (Decimal.new "13E80") . should_equal (Decimal.new "12E73")
+            Decimal.new "-12E73" . min (Decimal.new "-13E60") . should_equal (Decimal.new "-12E73")
+            Decimal.new "-12E73" . min (Decimal.new "-13E80") . should_equal (Decimal.new "-13E80")
+
+            Decimal.new "12" . max (Decimal.new "13") . should_equal (Decimal.new "13")
+            Decimal.new "12" . max (Decimal.new "11") . should_equal (Decimal.new "12")
+            Decimal.new "-12" . max (Decimal.new "-13") . should_equal (Decimal.new "-12")
+            Decimal.new "-12" . max (Decimal.new "-11") . should_equal (Decimal.new "-11")
+
+            Decimal.new "12.1" . max (Decimal.new "12.3") . should_equal (Decimal.new "12.3")
+            Decimal.new "12.2" . max (Decimal.new "11.7") . should_equal (Decimal.new "12.2")
+            Decimal.new "-12.1" . max (Decimal.new "-12.3") . should_equal (Decimal.new "-12.1")
+            Decimal.new "-12.2" . max (Decimal.new "-11.7") . should_equal (Decimal.new "-11.7")
+
+            Decimal.new "0" . max (Decimal.new "-1") . should_equal (Decimal.new "0")
+            Decimal.new "0" . max (Decimal.new "1") . should_equal (Decimal.new "1")
+            Decimal.new "-1" . max (Decimal.new "0") . should_equal (Decimal.new "0")
+            Decimal.new "1" . max (Decimal.new "0") . should_equal (Decimal.new "1")
+
+            Decimal.new "12E73" . max (Decimal.new "13E60") . should_equal (Decimal.new "12E73")
+            Decimal.new "12E73" . max (Decimal.new "13E80") . should_equal (Decimal.new "13E80")
+            Decimal.new "-12E73" . max (Decimal.new "-13E60") . should_equal (Decimal.new "-13E60")
+            Decimal.new "-12E73" . max (Decimal.new "-13E80") . should_equal (Decimal.new "-12E73")
```

Comment:
There are no tests for cases where decimals and non-decimals are intertwined.

I think both
```
Decimal.new "123" . max 100 . should_equal 123
124 . max (Decimal.new "101") . should_equal 124
```
would be good to check.

Reply:
Done